libs/utils/android/workload.py

```diff
@@ -94,11 +94,15 @@ def tracingStart(self):
             raise ValueError(msg)
         # Start FTrace
         if 'ftrace' in self.collect:
+            # Ensure symbols addresses are exposed via /proc/kallsyms
+            self._target.write_value('/proc/sys/kernel/kptr_restrict', 0)
```

Comment:
If adb is not running as root, then it will fail to write. Shouldn't be using target.execute() with as_root=True, or check Target.is_rooted before writing?